### ISSUE TITLE
Uncaught TypeError: Cannot read property 'buffer' of undefined

### ISSUE THREAD

**Issue by Bioblaze** [HelloWorld017/atom-discord#41]
[Enter steps to reproduce:]

1. Try and move another File to a Different Spot on the Screen So they are Side by Side

**Atom**: 1.23.3 x64
**Electron**: 1.6.15
**OS**: Microsoft Windows 10 Pro Education
**Thrown From**: [atom-discord](https://github.com/HelloWorld017/atom-discord) package 1.4.0

### Stack Trace

Uncaught TypeError: Cannot read property 'buffer' of undefined

```
At C:\Users\biobl\.atom\packages\atom-discord\src\index.js:71

TypeError: Cannot read property 'buffer' of undefined
    at atom.workspace.onDidChangeActiveTextEditor (/packages/atom-discord/src/index.js:71:78)
    at Function.module.exports.Emitter.simpleDispatch (~/AppData/Local/atom/app-1.23.3/resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (~/AppData/Local/atom/app-1.23.3/resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at Workspace.didChangeActivePaneItemOnPaneContainer (~/AppData/Local/atom/app-1.23.3/resources/app/src/workspace.js:501:30)
    at WorkspaceCenter.paneContainer.onDidChangeActivePaneItem (~/AppData/Local/atom/app-1.23.3/resources/app/src/workspace-center.js:17:20)
    at Function.module.exports.Emitter.simpleDispatch (~/AppData/Local/atom/app-1.23.3/resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (~/AppData/Local/atom/app-1.23.3/resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at PaneContainer.didChangeActiveItemOnPane (~/AppData/Local/atom/app-1.23.3/resources/app/src/pane-container.js:284:26)
    at PaneContainer.didActivatePane (~/AppData/Local/atom/app-1.23.3/resources/app/src/pane-container.js:262:18)
    at Pane.activate (~/AppData/Local/atom/app-1.23.3/resources/app/src/pane.js:1002:46)
    at Pane.split (~/AppData/Local/atom/app-1.23.3/resources/app/src/pane.js:1116:19)
    at Pane.splitRight (~/AppData/Local/atom/app-1.23.3/resources/app/src/pane.js:1059:23)
    at Object.<anonymous> (~/AppData/Local/atom/app-1.23.3/resources/app/node_modules/tabs/lib/layout.js:50:33)
    at Object.end (~/AppData/Local/atom/app-1.23.3/resources/app/node_modules/tabs/lib/layout.js:56:16)
    at HTMLLIElement.TabView.element.ondragend (~/AppData/Local/atom/app-1.23.3/resources/app/node_modules/tabs/lib/tab-view.js:61:29)
```

### Commands

```
  2x -8:36.4.0 core:move-left (input.hidden-input)
     -8:33 core:move-right (input.hidden-input)
  2x -8:28 core:copy (input.hidden-input)
     -8:19.4.0 core:paste (input.hidden-input)
  2x -8:03.5.0 core:copy (input.hidden-input)
     -7:59.8.0 core:paste (input.hidden-input)
  5x -7:59.3.0 core:move-up (input.hidden-input)
     -7:58.2.0 core:move-left (input.hidden-input)
     -7:57.9.0 core:move-up (input.hidden-input)
  3x -7:57.7.0 core:move-left (input.hidden-input)
     -7:56.8.0 core:move-right (input.hidden-input)
  9x -7:56.5.0 core:backspace (input.hidden-input)
     -7:54.7.0 core:paste (input.hidden-input)
  2x -4:29.4.0 core:backspace (input.hidden-input)
     -0:32.9.0 editor:newline (input.hidden-input)
  7x -0:27.3.0 core:backspace (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.30.9 
atom-bootstrap3 1.2.12 
atom-discord 1.4.0 
atom-ternjs 0.18.3 
autoclose-html 0.23.0 
autocomplete-modules 1.10.0 
file-icons 2.1.16 
highlight-selected 0.13.1 
language-ejs 0.4.0 
minimap 4.29.7 
minimap-find-and-replace 4.5.2 
minimap-git-diff 4.3.1 
minimap-highlight-selected 4.6.1 
minimap-pigments 0.2.2 
node-debugger 1.10.1 
npm-init 1.1.0 
open_in_cmd 0.6.1 
pretty-json 1.6.4 
wakatime 7.0.4 
```

**Comment by Bioblaze** [comment]
I'll work on fixing this. Just give me a bit of reading is all. And i'll figure it out.

**Comment by Bioblaze** [comment]
#42 Fixes this Issue. It also allows you to Support Split Window Files, by showing the correct window's information and project. <3

**Comment by HelloWorld017** [comment]
Closed by #42
